resources/config.rb

```diff
@@ -23,10 +23,10 @@
 property :config, Hash, required: true
 
 action :create do
-  require 'toml'
+  require 'toml-rb'
```

Comment:
I think this is wrong - this gem should still `require toml` which is kind of annoying.